lib/MusicBrainz/Server/Entity/URL/HMikuWiki.pm

```diff
@@ -5,7 +5,7 @@ use Moose;
 extends 'MusicBrainz::Server::Entity::URL';
 with 'MusicBrainz::Server::Entity::URL::Sidebar';
 
-sub sidebar_name { 'Miku Hatsune Wiki' }
+sub sidebar_name { 'Hatsune Miku Wiki' }
```

Comment:
See b6f8073.

Reply:
@julian45 wrote:
> See [b6f8073](https://github.com/metabrainz/musicbrainz-server/commit/b6f80734a8381e28d1cbf2b5f6de08f3f24252f8).

Thank you for having explained your change, but most of the arguments don’t stand. A _literal_ translation has _low-fidelity_ to the meaning even though it is pronounced in the same order. The official translation is normally always prevailing but this is a small enough change with still some benefits.

Reply:
@reosarevok wrote:
> maybe it'd be best to just show the Japanese name

Just showing Japanese characters is an anti-pattern with respect to internationalization. The base language for the MusicBrainz website is English and the targeted audience is not limited to native speakers. They can visually identify characters from the Latin alphabet but not necessarily characters from other writing systems. It would prevent them from visually identifying the name in the sidebar.

> We already show Japanese only for Japameta for example.

This isn’t a relevant example. There are only 11 "Japameta" URLs in the database. I rather wonder why there is this kind of unused code in the first place, and why it is named “Japameta” in the code and in STYLE-308 but not on their own website.

Even though there are 10 other URL handlers showing just non-Latin characters in the sidebar, no reason has ever been provided for doing it so, while internationalization has most often not been taken into account.